src/features/transfer-ownership/FormSteps/Details.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react';
+
+import { FormInputs, FormDetails } from '../../ui';
+
+interface TransferOwnershipProps {
```

Comment:
nit: I think this interface name could be clearer - the feature is called `transfer-ownership` which initially made me think `TransferOwnershipProps` was related to the feature as a whole, rather than the `Details` step.

Reply:
Ah good spot, no idea why I called it ```TransferOwnershipProps``` 😕 I (normally) copy the component/function name and add ```props``` to the end for interface/types like these

Reply:
updated to ```DetailsProps```